pisi/util.py

```diff
@@ -611,7 +610,7 @@ def save_elf_debug(f, o):
 def partition_freespace(directory):
     """Return free space of given directory's partition."""
     st = os.statvfs(directory)
-    return st[statvfs.F_BSIZE] * st[statvfs.F_BFREE]
+    return st.f_frsize * st.f_bfree
```

Comment:
Why `f_frsize` instead of `f_bsize`? `f_bfree` states: "Number of free blocks", and in fact `f_bsize` is the block size.